docs/hub/spaces.md

```diff
@@ -169,8 +169,26 @@ jobs:
 
 ## Troubleshoot
 
+### Streamlit
+
 Issues may occur when you use an unsupported Streamlit version. The Streamlit version is not configured in the **requirements.txt** file but rather in the `YAML` settings through the `sdk_version` setting. Not all Streamlit versions are supported. Check that you are using a supported version of Streamlit. Refer to the [reference section](#reference) for more information about supported versions.
 
+### Custom Python Spaces
+
+While not an official workflow yet, you are able to run your own Python + interface stack in Spaces by selecting Gradio as your SDK and serving a frontend on port `7680`. See the [templates](https://huggingface.co/templates#spaces) for examples.
+
+Because you are now responsible for the interface, you must add an [iFrame Resizer script](https://cdnjs.com/libraries/iframe-resizer) that is normally imported by the Gradio library:
```

Comment:
can you qualify in which cases this is needed? (i haven't needed it yet on my side) like if you want a fluid vertical layout, etc?

Reply:
👍 Have changed wording to be more clear about why it's needed

And that's right, needed essentially whenever the content exceeds the viewport height.

I just created a duplicate of one of the templates here as example, where the only change is iFrame Resizer being commented out: https://huggingface.co/spaces/ronvolutional/iframe-test
Compared to: https://huggingface.co/spaces/ronvolutional/http-server

 I tested Gradio HTML ouput and static SDK space before and they do not need it.

Reply:
makes sense, and i think we can even link to your example Spaces, those are a great explanation

Reply:
(made some suggestions)

Reply:
Nice additions. Just made the example suggestion more concise. Will merge when Charles/Christophe has a look!